### ISSUE TITLE
splot2model hangs if constraint list is empty

### ISSUE THREAD

**Issue by h3ssto** [davidfa71/Extending-Logic#2]
Example (generated with S.P.L.O.T.)
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<feature_model name="FM10-3CNF-FM-10-0-0.00-SAT-1">
<feature_tree>
:r R(R)
	:g _Gi__1(_Gi__1) [1,*] 
		: g_1_1(g_1_1)
		: g_1_2(g_1_2)
	:g _Gi__2(_Gi__2) [1,*] 
		: g_2_1(g_2_1)
		: g_2_2(g_2_2)
	:m m_3(m_3)
		:o o_3_1(o_3_1)
	:m m_4(m_4)
	:o o_5(o_5)
	:o o_6(o_6)
</feature_tree>
<constraints>
</constraints>
</feature_model>
<!--
*************************************************************
FEATURE TREE STATISTICS
  Model name.: FM10-3CNF-FM-10-0-0.00-SAT-1
  tree depth.: 2
  #root......: 1
  #opt.......: 3
  #man.......: 2
  #grouped...: 4
  #grouped [1,1]: 0
  #grouped [1,n]: 4
  #fgrp [1,1]: 0 (exclusive-OR)
  #fgrp [1,N]: 2 (inclusive-OR)
  #total features: 10
  #total nodes: 12
*************************************************************
CROSS-TREE CONSTRAINTS (Random 3-CNF Formula)
  CTC Representativeness (CTCR): 0.0%
  Number of 3-CNF clauses......: 0
  CTC clause density specified.: 0.00
*************************************************************
-->
```

**Comment by davidfa71** [comment]
That should be fixed now. Regards

**Comment by h3ssto** [comment]
I can confirm that the bug is fixed by 0107745251bf54f570766747d5da957b6157434e . Many thanks for your swift action!
